src/app/pages/MediaArticlePage/index.stories.tsx

```diff
@@ -4,20 +4,19 @@ import { withKnobs } from '@storybook/addon-knobs';
 import { ToggleContextProvider } from '#contexts/ToggleContext';
 import { RequestContextProvider } from '#contexts/RequestContext';
 import { UserContextProvider } from '#contexts/UserContext';
-import { ServiceContextProvider } from '../../contexts/ServiceContext';
 import { MEDIA_ARTICLE_PAGE } from '#app/routes/utils/pageTypes';
 import articleData from '#data/hausa/articles/cw43vy8zdjvo.json';
-import { ARTICLE_PAGE } from '#app/routes/utils/pageTypes';
 import tamilArticle from '#data/tamil/articles/c84m2jl4dpzo.json';
 import pidginArticle from '#data/pidgin/articles/cw0x29n2pvqo.json';
 import withPageWrapper from '#containers/PageHandlers/withPageWrapper';
 import withOptimizelyProvider from '#containers/PageHandlers/withOptimizelyProvider';
+import { ServiceContextProvider } from '../../contexts/ServiceContext';
 import MediaArticlePageComponent from './MediaArticlePage';
 
 const PageWithOptimizely = withOptimizelyProvider(MediaArticlePageComponent);
 const Page = withPageWrapper(PageWithOptimizely);
 
-// eslint-disable-next-line react/prop-types
+// @ts-expect-error - passing in partial data
```

Comment:
Nice, I like this syntax instead of `//@ ts-ignore` - even though it's "test" code, it explains why the type errors can be ignored.